geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml

```diff
@@ -530,102 +530,103 @@ vars:
       - name: searchAlias
         type: list
         description: >
-          Comma separated list of search aliases (keywords) to be added to the tsearch.ts field for
-          the considered tree item (theme, layer group or layer), that is to say that considered
-          tree item will match when searching for the search aliases.
+          Comma-separated list of search aliases (keywords) to be added to the tsearch.ts field for
+          the specified tree item (theme, layer group or layer). The tree item will match when
```

Comment:
The new texts are indeed better than the existing texts.
I am just wondering about translations, does this mean we have to redo translations for these texts?

Reply:
Good question, I will follow that :-)